tests/unit/setup-application-context-test.js

```diff
@@ -127,4 +135,11 @@ module('setupApplicationContext', function(hooks) {
   test('bubbles up errors', function(assert) {
     assert.rejects(visit('/widgets'), /Model hook error from \/widgets/);
   });
+
+  test('`transitionAborted` errors are not treated as test errors', async function(assert) {
+    await visit('/purgatory');
+    await visit('/');
+
+    assert.equal(currentURL(), '/purgatory');
```

Comment:
I might miss something obvious, but why is the current URL expected to be `/purgatory` here, after having visited `/`?